flink-fs-tests/src/test/java/org/apache/flink/hdfstests/DistributedCacheDfsTest.java

```diff
@@ -128,7 +130,7 @@ public void testDistributeFileViaDFS() throws Exception {
 
 		env.fromElements(1)
 			.map(new TestMapFunction())
-			.print();
+			.addSink(new DiscardingSink<>());
```

Comment:
probably cleaner to address this issue in a separate commit as the relation isn't obvious.